bin/makeDocs

```diff
@@ -1,14 +1,19 @@
 #!/usr/bin/env python
 #
 from __future__ import with_statement
+from __future__ import print_function
 
-import fnmatch, tempfile
+import tempfile
 import hashlib
-import os, pwd, re, sys
+import os
+import pwd
+import re
+import sys
 import eups
 
 sep = "\a"                              # field separator (as values can contain whitespace)
-ignoredProducts = ["implicitProducts",]
+ignoredProducts = ["implicitProducts", ]
```

Comment:
Good point. It's only needed for tuples and I should have removed it rather than simply fixing the PEP-8 issue. There is a repeat later that I will also fix.